spec/rubocop/cop/rails/save_bang_spec.rb

```diff
@@ -0,0 +1,69 @@
+# encoding: utf-8
+# frozen_string_literal: true
+require 'spec_helper'
+
+describe RuboCop::Cop::Rails::SaveBang do
+  subject(:cop) { described_class.new }
+
+  shared_examples 'checks_offense' do |method|
+    it "when using #{method} with arguments" do
+      inspect_source(cop, "object.#{method}(name: 'Tom', age: 20)")
+
+      if method == :destroy
+        expect(cop.messages).to be_empty
+      else
+        expect(cop.messages)
+          .to eq([["Use #{method}! instead of #{method} ",
+                   'if the return value is not checked.'].join])
+      end
+    end
+
+    it "when using #{method} without arguments" do
+      inspect_source(cop, method.to_s)
+
+      expect(cop.messages)
+        .to eq([["Use #{method}! instead of #{method} ",
+                 'if the return value is not checked.'].join])
+    end
+
+    it "when using #{method}!" do
+      inspect_source(cop, "object.#{method}!")
+
+      expect(cop.messages).to be_empty
+    end
+
+    it "when using #{method} with 2 arguments" do
+      inspect_source(cop, "Model.#{method}(1, name: 'Tom')")
+
+      expect(cop.messages).to be_empty
+    end
+
+    it "when using #{method} with wrong argument" do
+      inspect_source(cop, "object.#{method}('Tom')")
+
+      expect(cop.messages).to be_empty
+    end
+
+    it "when using #{method} with receiver" do
```

Comment:
`object` is the receiver. How about "when assigning the return value of #{method}"?